src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/MathHelpers.cs

```diff
@@ -28,7 +30,7 @@ private static UInt64 Mul32x32To64(UInt32 a, UInt32 b)
         }
 
         // Helper to get high 32-bit of 64-bit int
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [MethodImpl(﻿MethodImplOptions.AggressiveInlining)]
```

Comment:
Could you please undo the change to this line?

It shows diff because of there is a weird Unicode character (EF BB BF) that the GitHub UI does not show.